anchore_engine/services/policy_engine/__init__.py

```diff
@@ -157,6 +157,7 @@ def _init_distro_mappings():
         DistroMapping(from_distro="rhel", to_distro="rhel", flavor="RHEL"),
         DistroMapping(from_distro="ubuntu", to_distro="ubuntu", flavor="DEB"),
         DistroMapping(from_distro="amzn", to_distro="amzn", flavor="RHEL"),
+        DistroMapping(from_distro="redhat", to_distro="rhel", flavor="RHEL"),
```

Comment:
This looks like it is causing both "rhel" and "redhat" to be available when `DistroNamespace()` is called. If that is intended, then it seems the the unit tests are straightforward to fix:

```
r7 = DistroNamespace(name="rhel", version="7", like_distro="rhel")
>       assert set(r7.mapped_names()) == {"centos", "fedora", "rhel"}
E       AssertionError: assert {'centos', 'f...dhat', 'rhel'} == {'centos', 'fedora', 'rhel'}
E         Extra items in the left set:
E         'redhat'
E         Full diff:
E         - {'centos', 'fedora', 'rhel'}
E         + {'centos', 'fedora', 'rhel', 'redhat'}
E         ?                            ++++++++++
```

Reply:
ack, @alfredodeza yep, test fix just added

Reply:
thanks, btw